tests/src/test_cases/widgets/test_bar.c

```diff
@@ -399,4 +403,92 @@ void test_bar_render_corner(void)
     render_test_screen_create(true, LV_GRAD_DIR_VER, "widgets/bar_corner_6.png");
 }
 
+void test_bar_orientation(void)
```

Comment:
It would be good to **add reference image** also to compare the result with that.

Please simplify the code in this test, it just needs to test `lv_bar_set_orientation` function, and no need for special style for bar. Except maybe: add gradient color to see that orientation really works as expected.

Reply:
I agree with these too.